commands/command_filter_process.go

```diff
@@ -203,7 +203,7 @@ func filterCommand(cmd *cobra.Command, args []string) {
 	}
 
 	if len(malformed) > 0 {
-		fmt.Fprintf(os.Stderr, "Encountered %d file(s) that should have been pointers, but weren't:\n", len(malformed))
+		fmt.Fprintf(os.Stderr, "Encountered %d file(s) that should have been Git LFS pointers, but they're Git files:\n", len(malformed))
```

Comment:
This is definitely more detailed, but it's also more than 80 characters long, which means it will wrap on some screens.  Can we shorten it?  Maybe `Encountered %d file(s) that should have been Git LFS pointers, but weren't`?  Or something else?

Reply:
Your suggestion is definitely better as it will fit in one line. However I'm afraid it might not be clear enough. I'm now getting inspiration from the next `if` condition, which displays an initial error message, then loops on a collection, then display a help message with a `git lfs` command to get more details. I think we could do the same: an initial (short) but a bit clearer output message, and then instructions on what to do next.